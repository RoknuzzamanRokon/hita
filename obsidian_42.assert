type: account-key
authority-id: canonical
revision: 2
public-key-sha3-384: BWDEoaqyr25nF5SNCvEv2v7QnM9QsfCc0PBMYD_i2NGSQ32EF2d4D0hqUel3m8ul
account-id: canonical
name: store
since: 2016-04-01T00:00:00.0Z
body-length: 717
sign-key-sha3-384: -CvQKAwRQ5h3Ffn10FILJoEZUXOv6km9FwA80-Rcj-f-6jadQ89VRswHNiEB9Lxk

AcbBTQRWhcGAARAA0KKYYQWuHOrsFVi4p4l7ZzSvX7kLgJFFeFgOkzdWKBTHEnsMKjl5mefFe9ji
qe8NlmJdfY7BenP7XeBtwKp700H/t9lLrZbpTNAPHXYxEWFJp5bPqIcJYBZ+29oLVLN1Tc5X482R
vCiDqL8+pPYqBrK2fNlyPlNNSum9wI70rDDL4r6FVvr+osTnGejibdV8JphWX+lrSQDnRSdM8KJi
UM43vTgLGTi9W54oRhsA2OFexRfRksTrnqGoonCjqX5wO3OFSaMDzMsO2MJ/hPfLgDqw53qjzuKL
Iec9OL3k5basvu2cj5u9tKwVFDsCKK2GbKUsWWpx2KTpOifmhmiAbzkTHbH9KaoMS7p0kJwhTQGA
o9aJ9VMTWHJc/NCBx7eu451u6d46sBPCXS/OMUh2766fQmoRtO1OwCTxsRKG2kkjbMn54UdFULl9
VfzvyghMNRKIezsEkmM8wueTqGUGZWa6CEZqZKwhe/PROxOPYzqtDH18XZknbU1n5lNb7vNfem9F
2ai+3+JyFnW9UhfvpVF7gzAgdyCqNli4C6BIN43uwoS8HkykocZS/+Gv52aUQ/NZ8BKOHLw+7ant
Q0o8W9ltSLZbEMxFIPSN0stiZlkXAp6DLyvh1Y4wXSynDjUondTpej2fSvSlCz/W5v5V7qA4nIcG
vUvV7RjVzv17ut0AEQEAAQ==

AcLDXAQAAQoABgUCV83k9QAKCRDUpVvql9g3IBT8IACKZ7XpiBZ3W4lqbPssY6On81WmxQLtvsMV
WTp6zZpl/wWOSt2vMNUk9pvcmrNq1jG9CuhDfWFLGXEjcrrmVkN3YuCOajMSPFCGrxsIBLSRt/bP
nrKykdLAAzMfG8rP1d82bjFFiIieE+urQ0Kcv09Jtdvavq3JT1Tek5mFyyfhHNlQEKOzWqmRWiLg
3c3VOZUs1ZD8TSlnuq/x+5T0X0YtOyGjSlVxk7UybbyMNd6MZfNaMpIG4x+mxD3KHFtBAC7O6kLe
eX3i6j5nCY5UABfA3DZEAkWP4zlmdBEOvZ9t293NaDdOpzsUHRkoi0Zez/9BHQ/kwx/uNc2WqrYm
inCmu16JGNeXqsyinnLl7Ghn2RwhvDMlLxF6RTx8xdx1yk6p3PBTwhZMUvuZGjUtN/AG8BmVJQ19
rsGSRkkSywvnhVJRB2sudnrMBmNS2goJbzSbmJnOlBrd2WsV0T9SgNMWZBiov3LvU4o2SmAb6b+k
rYwh8H5QHcuuYJuxDjFhPswIp6Wes5T6hUicf3SWtObcDS4HSkVS4ImBjjX9YgCuFy7QdnooOWEY
aPvkRw3XCVeYq0K6w9GRsk1YFErD4XmXXZjDYY650MX9v42Sz5MmphHV8jdIY5ssbadwFSe2rCQI
6UX08zy7RsIb19hTndE6ncvSNDChUR9eEnCm73eYaWTWTnq1cxdVP/s52r8uss++OYOkPWqh5nOu
haRn7INjH/yZX4qXjNXlTjo0PnHH0q08vNKDwLhxS+D9du+70FeacXFyLIbcWllSbJ7DmbumGpFo
yYbtj3FDDPzachFQdIG3lSt+cSUGeyfSs6wVtc3cIPka/2Urx7RprfmoWSI6+a5NcLdj0u2z8O96
HxeIgxDpg/3gT8ZIuFKePMcLDM19Fh/p0ysCsX+84B9chNWtsMSmIaE57V+959MVtsLu7SLb9gi7
skrju0pQCwsu2wHMLTNd1f3PTHmrr49hxetTus07HSQUApMtAGKzQilF5zqFjbyaTd4xgQbd+PKW
CjFyzQTDOcUhXpuUGt/IzlqiFfsCsmbj2K4KdSNYMlqIgZ3Azu8KvZLIhsyN7v5vNIZSPfEbjdeu
ClU9r0VRiJmtYBUjcSghD9LWn+yRLwOxhfQVjm0cBwIt5R/yPF/qC76yIVuWUtM5Y2/zJR1J8OFq
qWchvlImHtvDzS9FQeLyzJAOjvZ2CnWp2gILgUz0WQdOk1Dq8ax7KS9BQ42zxw9EZAEPw3PEFqRy
IQsRTONp+iVS8YxSmoYZjDlCgRMWUmawez/Fv5b9Fb/XkO5Eq4e+KfrpUujXItaipb+tV8h5v3tr
oG3Ie3WOHrVjCLXIdYslpL1O4nadqR6Xv58pHj6k

type: account
authority-id: canonical
account-id: WfRJPqOpcF44N2eXtz8wK4LhCx7r7eHu
display-name: Obsidian
timestamp: 2022-10-12T15:33:52.926955Z
username: obsidianmd
validation: unproven
sign-key-sha3-384: BWDEoaqyr25nF5SNCvEv2v7QnM9QsfCc0PBMYD_i2NGSQ32EF2d4D0hqUel3m8ul

AcLBUgQAAQoABgUCY0beYAAA4swQAEvLCKz2A0kyCDVmIOBxCHZUXB/PN6kNzgCcU52rt/hZPHQ0
t6A38MtHbKtt7+CAhH6uLYorxBttC3NGCTCfLVOMlxgvdCZyF5TqgtoLLb2LvKdPFdUv/GRb5bpF
zlMdvJhVSk/Q2hBkCnGuXX63P0SgMtqJHgG/UvL3pAdSZjSQ/WpAEHZ3yo2842DaxwgKTBW2tSjt
5OdH8c9cbJJiVTFVaUMLrnxJWvItPrYi4vFsbv5+ZNhe+7tYxUvNMNcJSZBfKdHkqy+J0H9ff4Xr
CQtdCH7qDwlYl0wIZ7eB4153hxB8GP2r/Meos0wvabYnGrNw/IjamXFnlSVJd/M1PkjaQl5ETVb6
Y3kMaXVyI4Q6yTY2zA04I0gbFI56hUV80bOLJTgtMCWr40ZNDRosYJOg8LbMkLyfF2ceLSHWLSBg
PgDFZNcAQvaK2FKaFpNcpr+4BGPgHE8rKrKJdTrlcE3DS4A9FzdzHcR/GbSozUTAVvWxBdWNEG3g
7SorXVDzYdCGQTlZSkmiaGeuWiGyxR3VNHf6iTs+jc4047Z/9xMhKMONoZz4KD7j0aB946lShocZ
VpwIPCsmtfX2gSD/d1J2bK4uCVKtmQvYxbV3LnaUPpoVG43y12ERy/QMEiIaEaQaEoOP5Bv0vn6N
D1MRwT9afW61Obxj0RWX0UGrZDWW

type: snap-declaration
authority-id: canonical
series: 16
snap-id: XSpRD8poWONFIT1z9TleS4XrS1u0MlfG
publisher-id: WfRJPqOpcF44N2eXtz8wK4LhCx7r7eHu
snap-name: obsidian
timestamp: 2022-10-12T16:37:21.654644Z
sign-key-sha3-384: BWDEoaqyr25nF5SNCvEv2v7QnM9QsfCc0PBMYD_i2NGSQ32EF2d4D0hqUel3m8ul

AcLBUgQAAQoABgUCY0btQQAAjl0QABthMoNPSeMmQKpn1nMrjNAQ730+16XbspS/THXCkBUbNcPp
TY4oZX1A3dUXwtEJO2iuZlvoLcDjb21II4068xly6oiTGq2gfOKajQJvd50aDnyXEXxDUlK/Ot5g
KTpD5DhvooBemWLkVKjehcSdV9DQ7QD021k8YfQmvlLMXCBKbA2CaWXPKbSUVWwSAVTsv1IpDo0n
Of1ueNpHe2m3VKl+2VjEf5MbD3bhYRI9qxt/sPM5ryaOtjmi1XnwuEqGlf7caqZJvmWykzezBRd4
lUkumNGyKUdcFX3aqn/H/76NbV7MzUQPzEOaF/q9+SD6KYdjSdsydWl3RreXs1iAmuRAj5ACs9QO
JwvXvZAvRNlRGeHBfOoRpZ9eRIRNsBrFPFId6Lv9RjVT3WYr9A5ZU7F/vgiuD6jJh1hiGFpkK8u3
s2FsCwyUz1dfxwYMfIdnCq8PF+xOmt8PJWenjLGdpVEgU3iVBBjlWUf+B/JBI4bDFED3XyhUXAWB
zBL3JlA8N02RYfV8UDFoOZrvfVpouzHlGj48GdR4sLe+UbiK0zSJixPv3lHe/ZVkCXksv3kzKbui
sM+QEbFhxJLctCflm888TEdROls79jyeGOBOLaTikK2feqICO7fEi3myGKmaHdKqgmXAEv0r2FOF
SmmRsOGHLgJlLFZo/RGS2za4Jp35

type: snap-revision
authority-id: canonical
snap-sha3-384: SZP6wmTZyP1lDYNnMFGhYA6IPmtREHaxP3NXpdKbev5kZntfGFLtgmZe_t61OkRa
developer-id: WfRJPqOpcF44N2eXtz8wK4LhCx7r7eHu
provenance: global-upload
snap-id: XSpRD8poWONFIT1z9TleS4XrS1u0MlfG
snap-revision: 42
snap-size: 126550016
timestamp: 2025-01-30T22:55:46.587757Z
sign-key-sha3-384: BWDEoaqyr25nF5SNCvEv2v7QnM9QsfCc0PBMYD_i2NGSQ32EF2d4D0hqUel3m8ul

AcLBUgQAAQoABgUCZ5wDcgAA6eMQAAqmRQmPwX2L7eo1HlSSBmafFJcmgxVRuTfZNKGafctyMPjJ
A5SkD+vXwKx2GSXXxLjEwKfOX5+lKbrlNJoYtGpJZN9rguzaJ77NFrOUdxkuxmWvjMW/IeexU0CU
iLZbtdcVvdtMTvc8+5pPQq01+gC0kdnkiBLHi9LDT06bIK/YWcJ7C9lzJLUBY2B9YZ6LKrk/7wJh
V8dDnNEyBiyKv6efhEZCY0v7n2KOachkkUdxxVggdSAvySMFSSpxfn2ImVhT8/BR1AwE2Z3CmN9Z
koBchjt/98jHGUXKhJrBJI2uPeJ9c4SLcWMCMyudEwXqym466ViUusmbx8KSHIGswDVcM6wcDn7t
mktBcTBP1COpILC2naK+/LILjtDLOVpWfp5NyfP109DcMpqGr1TEX1Iue2xNmPIu4ml2gtyau57V
LejauJRSJG9wQLcZcv2NNNCxKQ044BN5JXjBTau3IMvoHHHP4zaZ/Vzs4sWT0sA9nVcY421bf4n1
j/dHxt8Ke4X6n0GCxdBmuU5JBrEXLRzj3SlM3sldzG8oAXAiKq0enleXikuRnvA3icdELL+ZBp78
juxEUS6a0k6D4b+UfCnyqeQ4ovQb422cbcnwiQLVER4n8a9VJuBMY7avXtX2tR7fYsTqdkUUoRqW
laDm5bT2f2i7my4+Ix5h7oFy0ofh
