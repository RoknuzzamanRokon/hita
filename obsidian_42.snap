version https://git-lfs.github.com/spec/v1
oid sha256:0aa4d9e5317e5b891f847746ecf5887a1e2992185455426df8d8669f1af8ce86
size 126550016
